lib/Elastica/Transport/AbstractTransport.php

```
@@ -99,14 +99,13 @@ public static function create($transport, Connection $connection, array $params
             } else {
                 $transport = ucfirst($transport);
             }
-
-            $className = 'Elastica\\Transport\\'.$transport;
-
-            if (!class_exists($className)) {
```

we should still throw an invalid exception in case none of the classes matches.